hotsos/client.py

```diff
@@ -184,12 +185,27 @@ def _save_to_file(path, content):
             fd.write(content)
             fd.write("\n")
 
+    @staticmethod
+    def compress(path):
+        """
+        Compress path to path.gz and delete path.
+        """
+        cpath = path + '.gz'
+        with open(path, 'rb') as fd_in:
+            with gzip.open(cpath, 'wb') as fd_out:
+                shutil.copyfileobj(fd_in, fd_out)
+
+        os.remove(path)
```

Comment:
Is it still OK to delete even if compression failed for some reason?

Reply:
that would throw an exception i presume so it would not get that far